packages/cozy-client/src/CozyClient.spec.js

```diff
@@ -84,6 +84,23 @@ describe('CozyClient initialization', () => {
     expect(client.stackClient.token.token).toBe(token)
   })
 
+  it('can be instantiated from environment with OAuth token', () => {
+    const url = 'https://testcozy.mycozy.cloud'
+    const token = 'test-token'
+    const creds = {
+      token: {
+        accessToken: token
+      }
+    }
+    global.process.env.COZY_URL = url
+    global.process.env.COZY_CREDENTIALS = JSON.stringify(creds)
+    global.process.env.NODE_ENV = 'development'
+    client = CozyClient.fromEnv()
+    expect(client.stackClient.uri).toBe(url)
+    expect(client.stackClient.token.accessToken).toBe(token)
+    global.process.env.NODE_ENV = 'test'
```

Comment:
It is a bit unhygienic to let COZY_URL etc.. inside process.env after the test. Additionally, if one of the expect fails, the process.env.NODE_ENV is not set back to test.

I think you should use the second parameter env of fromEnv to mock the environment. 

```
client = CozyClient.fromEnv({ COZY_URL: 'blabla', COZY_CREDENTIALS: 'balbla' , NODE_ENV: 'bblabla' })
```